sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcSourceSuite.scala

```diff
@@ -62,6 +63,22 @@ class HiveOrcSourceSuite extends OrcSuite with TestHiveSingleton {
        """.stripMargin)
   }
 
+  test("SPARK-22972: hive orc source") {
+    spark.sql(
+      s"""CREATE TABLE normal_orc_as_source_hive
+         |USING org.apache.spark.sql.hive.orc
+         |OPTIONS (
+         |  PATH '${new File(orcTableAsDir.getAbsolutePath).toURI}'
+         |)
+       """.
+        stripMargin)
+    spark.sql(
+      "desc formatted normal_orc_as_source_hive").show()
```

Comment:
?

Reply:
change it to spark.sql("desc formatted normal_orc_as_source_hive").show(), is it ok?

How to get the warning and verify it in code?